mage_ai/frontend/oracle/components/Table/BaseTable.tsx

```diff
@@ -160,27 +150,21 @@ import { cutTextSize, getColumnWidth } from './helpers';
               return (
                 // eslint-disable-next-line react/jsx-key
                 <tr {...row.getRowProps()}>
-                  {row.cells.map(cell => (
+                  {row.cells.map((cell) => (
```

Comment:
Read the style guide. If you are returning something immediately in a loop without an explicit return function, you don't need to wrap a single parameter in parentheses

Reply:
Got it, will fix that. I forgot about it after I was using cell, i but removed the i.